### DISCUSSION TITLE
카카오맵에서 예약으로 넘어가지 않아요..

### DISCUSSION THREAD

**Discussion by jh8181** [changdoc/naver-vaccine-macro#230] [Q&A]
카카오맵에서 자동새로고침 으로 켜놓고, 

그후 핸드폰으론 알람이 오는데 

 pc 에서 켜놓은 카카오맵에서 아무 반응이 없어요.. 화살표만 돌아갈뿐.. 예약으로 넘어가지도 않고..그러네요..ㅠㅠ

머가 문제일까요...?

**Comment by changdoc** [comment]
https://github.com/changdoc/naver-vaccine-macro/issues/166
여기 마지막 내용 보시면 
카카오맵은 지도 화면상에 변화시키는 부분은 화살표 돌리기 시도(시도는 했으나 잘 돌아가진 않네요)만 동작하고 잔여 백신 조회만 백그라운드에서 계속 수행해서 잔여백신이 발생했는지 파악하여 잔여 수량이 발생했다고 판단되면 해당 병원 예약 신청 페이지로 이동 하게 동작하게 되어 있습니다.

개발자 도구 여시고 network탭에서 호출 기록이 계속해서 늘어나지 않거나 빨간색으로 에러가 발생하고 있다면 정상 동작 상황이 아니니 스크린샷이나 설명과 함께 제보해주시면 확인해 보도록 하겠습니다.

**Reply by jh8181** [reply]
덕분에 어제 잡았어요! 감사합니다!
